src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -451,4 +451,26 @@ class JavaMigrations {
       .insert()
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "14.ea.5-open", _))
   }
+
+  @ChangeSet(order = "169", id = "169-add_adoptopenjdk-hs_8_0_222", author = "jschalanda")
+  def migrate169(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_linux_hotspot_8u222b10.tar.gz", Linux64),
```

Comment:
We require a vendor to be added the end of every Java migration now. Could I ask you to add that to all your new versions please?